CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
-  | bright-red    | `#df3034` |           | removed - similar to the new red  |
-  | orange        | `#f47738` | `#f47738` |                                   |
-  | brown         | `#b58840` | `#b58840` |                                   |
-  | yellow        | `#ffbf47` | `#ffdd00` | updated                           |
-  | light-green   | `#85994b` | `#85994b` |                                   |
-  | green         | `#006435` | `#00703c` | updated                           |
-  | turquoise     | `#28a197` | `#28a197` |                                   |
-  | light-blue    | `#2b8cc4` | `#5694ca` | updated                           |
-  | blue          | `#005ea5` | `#1d70b8` | updated                           |
-  | dark-blue     |           | `#003078` | added                             |
-  | black         | `#0b0c0c` | `#0b0c0c` |                                   |
-  | grey-1        | `#6f777b` |           | removed                           |
-  | dark-grey     |           | `#6f777b` | added                             |
-  | grey-2        | `#bfc1c3` |           | removed                           |
-  | mid-grey      |           | `#b1b4b6` | added                             |
-  | grey-3        | `#dee0e2` |           | removed                           |
-  | grey-4        | `#f8f8f8` |           | removed                           |
-  | light-grey    |           | `#f3f2f1` | added                             |
-  | white         | `#ffffff` | `#ffffff` |                                   |
-
-  ([PR #1288](https://github.com/alphagov/govuk-frontend/pull/1288))
-
-- The button component now uses the green from the colour palette, instead of
-  a custom green used only for the button.
-
-  **Note:** If you are using [compatibility mode], the existing button colour
-  will be preserved.
-
-  ([PR #1288](https://github.com/alphagov/govuk-frontend/pull/1288))
-
-- The confirmation panel now uses a green background rather than a turquoise
-  background.
-
-  **Note:** If you are using [compatibility mode], the existing turquoise panel
-  colour will be preserved.
-
-  ([PR #1288](https://github.com/alphagov/govuk-frontend/pull/1288))
-
-- Links now get darker when hovered, rather than lighter.
-
-  **Note:** If you are using [compatibility mode], the existing link hover style
-  will be preserved.
+```scss
+@import "node_modules/govuk-frontend/govuk/all";
+```
+If you’ve [added `node_modules/govuk-frontend` as a Sass include path](https://github.com/alphagov/govuk-frontend/blob/268b1e1b962d4bd293c5681868a2119f8b4ece60/docs/installation/installing-with-npm.md#optional-resolving-scss-import-paths), add `govuk/` to your `@import` paths:
 
-  ([PR #1288](https://github.com/alphagov/govuk-frontend/pull/1288))
+```scss
+@import "govuk/all";
+```
 
-- Spacing of tabs list updated to be more inline with similar lists on GOV.UK and the Design System
+####  If you’re using Javascript
```

Comment:
Could we say "GOV.UK Frontend Javascript"

Reply:
I've added this to a list of potential post-release style changes.